src/tests/ftest/vmd/led.py

```diff
@@ -44,34 +43,58 @@ def get_nvme_device_ids(self):
         uuid = []
         for value in list(resp['host_storage_map'].values()):
             if value['storage']['smd_info']['devices']:
-                for target in range(self.targets):
-                    uuid.append(value['storage']['smd_info']['devices'][target]['uuid'])
+                total_uuid = len(value['storage']['smd_info']['devices'])
+                for device in range(total_uuid):
+                    uuid.append(value['storage']['smd_info']['devices'][device]['uuid'])
         return uuid
 
+    def run_vmd_led_identify(self, device_id=None):
+        """Run the VMD LED identify command.
+
+        Args:
+            device_id (str): Device UUID
+        Returns:
+            dmg LED identify command response.
+        """
+        if device_id is None:
+            self.fail("No device id provided")
+
+        try:
+            result = self.dmg.storage_led_identify(ids=device_id)
+        except CommandFailure as details:
+            self.fail("dmg command failed: {}".format(details))
+
+        self.log.info(result)
+        if result['error'] or len(result['response']['host_errors']) > 0:
+            if result['error']:
+                self.fail("dmg command failed: {}".format(result['error']))
+            else:
+                self.fail("dmg command failed: {}".format(result['response']['host_errors']))
```

Comment:
Recommend simplifying...
```suggestion
        if result['error']:
            self.fail("dmg command failed: {}".format(result['error']))
        elif len(result['response']['host_errors']) > 0:
            self.fail("dmg command failed: {}".format(result['response']['host_errors']))
```

Reply:
Done.